src/components/EditProfileForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Form, t } from '../components/Form';
+import Button from '../components/Button';
+import ProfileRequester from '../helpers/requesters/ProfileRequester'
+import BaseRequester from '../helpers/requesters/BaseRequester';
+
+class EditProfileForm {
+  constructor(props) {
+    super(props);
+
+    this._getInitialFormValues = this._getInitialFormValues.bind(this);
+    this._getFormType = this._getFormType.bind(this);
+    this._getFormOptions = this._getFormOptions.bind(this);
+    // this._clearFormErrors = this._clearFormErrors.bind(this);
+    this._onFormChange = this._onFormChange.bind(this);
+
+    this.state = {
+      formValues: this.getInitialFormValues(),
+      updating: true,
+      errors: [],
+    };
+  }
+
+  getInitialFormValues() {
+    return (
+      first_name: this.props.first_name,
+      last_name: this.props.last_name,
+      email: this.props.email,
+    );
+  }
+
+  getFormType() {
+    return t.struct({
+      firstName: t.String,
+      lastName: t.String,
+      email: t.String,
+    });
+  }
+
+  getFormOptions() {
+    return {
+      fields: {
+        firstName: {
+          hasError: !!this.state.errors.first_name,
+          error: this.state.errors.first_name,
+          label: 'First Name',
+        },
+        lastName: {
+          hasError: !!this.state.errors.last_name,
+          error: this.state.errors.last_name,
+          label: 'Last Name',
+        },
+        email: {
+          label: 'Email',
+          hasError: !!this.state.errors.email,
+          error: this.state.errors.email,
+        },
+      },
+    };
+  }
+
+  _onFormChange(values) {
+    this.setState({ formValues: values });
+  }
+
+  _handleSave() {
+    this.props.updateSave({current_veteran: this.state.formValues})
+  }
+
+  _renderSaveButton() {
+    return (
+      <Button
+        onPress = {this._handleSave()}
```

Comment:
you have to pass in the function here without the `()` so I would recommend binding `_handleSave` in the constructor and then pass in `this._handleSave` here